src/interfaces/tests/test_siease_states_interface.py

```diff
@@ -0,0 +1,25 @@
+from unittest import TestCase
+
+from mongoengine import connect, disconnect
+
+from src.interfaces import DistributionInterface
+from src.models.db.disease_states import DiseaseStates
+
+
+class DiseaseStatesTestCase(TestCase):
+    def setUp(self) -> None:
+        connect('mongoenginetest', host='mongomock://localhost', alias="DiseaseStatesInterfaceTestCase")
```

Comment:
* Has this 80 characters?
* Remember to set only single quote `'` or  double quote `"` for strings